server/db/migrations/20201020171509-add-type-column-to-bug.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('bugs', 'category', {
+      type: Sequelize.INTEGER,
+      defaultValue: 0,
```

Comment:
you didn't want to add not null here?

Also are we leaving it as 'type' rather than category?